File: java/src/main/java/com/example/todo/repository/Todo.java
package com.example.todo.repository;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;

@Entity
@Table(name = "todos")
@Getter
@Setter
@AllArgsConstructor()
public class Todo {
    @Id
    @GeneratedValue
    private int id;

    private Date createdAt;

    private Date updatedAt;

    private Date deadlineAt;

    private String title;

    private String description;

    private boolean isDone;

    protected Todo() { }
}

File: dotnet/Data/Models/Todo.cs
namespace Todo.Data.Models;

public class Todo
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public DateTime DeadlineAt { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public bool IsDone { get; set; }
}